File: gcd/standartway/gcd.py
import time
starta = 2
startb = 1
starttime = time.time()
while (starta <= 1000):
    while (startb <= starta):
        a = starta
        b = startb
        c = 1
        i = 1
        while(i <= startb):
            if(starta % i == 0 and startb % i == 0):
                c = i
            i += 1
        print(("ggT von a = " + str(starta) + " und b = " + str(startb) +
        " ist c =  " + str(c)))
        startb = startb + 1
    startb = 1
    print(("---------------------" + str(starta) + "-------------------------"))
    starta = starta + 1
endtime = time.time()

runtime = endtime - starttime
print((str(runtime) + " sec"))

File: gcd/standartway/gcd.lua
starta = 2
startb = 1
starttime = os.time()
while (starta <= 1000) do
    while (startb <= starta) do
        a = starta
        b = startb
        c = 1
        i = 1
        while(i <= startb) do
            if(starta % i == 0 and startb % i == 0) then
                c = i
              end
            i = i + 1
          end
        print("ggT von a = " .. starta .. " und b = " .. startb ..
        " ist c =  " .. c)
        startb = startb + 1
      end
    startb = 1
    print("---------------------" .. starta .. "-------------------------")
    starta = starta + 1
  end
endtime = os.time()

runtime = endtime - starttime
print(runtime .. " sec")
